src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -117,6 +123,9 @@
     
     private static final long DEFAULT_GUESTBOOK_RESPONSES_DISPLAY_LIMIT = 5000L; 
     
+    public final static String DEFAULTCURATIONLABELSET = "DEFAULT";
+    public final static String CURATIONLABELSDISABLED = "DISABLED";
```

Comment:
Do these English words appear in the UI anywhere? If so, let's make sure they can be translated.

Reply:
default isn't shown/there's already a bundle entry from when storage drivers were made configurable, but disabled hadn't been handled yet. Now done.